src/Atmos/Model/turbulence.jl

```diff
@@ -57,12 +57,19 @@ end
 struct SmagorinskyLilly{T} <: TurbulenceClosure
   C_smag::T
 end
-vars_aux(::SmagorinskyLilly,T) = @vars(Δ::T)
+vars_aux(::SmagorinskyLilly,T) = @vars(Δ::T, u_τ::T)
 vars_gradient(::SmagorinskyLilly,T) = @vars(θ_v::T)
 vars_diffusive(::SmagorinskyLilly,T) = @vars(∂θ∂Φ::T)
 function atmos_init_aux!(::SmagorinskyLilly, ::AtmosModel, aux::Vars, geom::LocalGeometry)
   aux.turbulence.Δ = lengthscale(geom)
 end
+function atmos_nodal_update_aux!(::SmagorinskyLilly, ::AtmosModel, state::Vars, diffusive::Vars, aux::Vars, t::Real)
+  ρ𝛕 = diffusive.ρτ
+  ρ = state.ρ
+  𝛕 = ρ𝛕 ./ ρ
+  @inbounds τ_w = sqrt(𝛕[1,3]^2 + 𝛕[2,3]^2)
+  aux.turbulence.u_τ = sqrt(τ_w / ρ)
```

Comment:
Does this need to be consistent with the state later? Updating in `update_aux!` will not guarantee this.

Reply:
If this is needed to be consistent, then I would suggest computing when it's needed.

Reply:
Noted - I placed this in there as an example of such a value requiring the diffusive state; computing this as necessary seems to be the solution since it needs to be consistent with the state. I believe I can access the diffusive state for visualisation through `param.diff` so this is fine for now.

Reply:
> I believe I can access the diffusive state for visualisation through `param.diff` so this is fine for now.

Yeah, that's what I'd suggest if you just need it for viz.